### PR TITLE
Add Uebergame as clone to ARMA series

### PR DESCRIPTION
**Pull request by Hiradur**

### CODE REVIEW

**Comment by Hiradur** [comment]
Please read https://github.com/piranha/osgameclones/pull/400#issuecomment-265736035 before merging.

File: games/a.yaml [hunk 1]
```diff
@@ -250,6 +250,32 @@
...
 
+- name: ARMA series
+  names:
+  - 'ARMA: Armed Assault'
+  - ARMA 2
+  - ARMA 3
+  - Operation Flashpoint
+  meta:
+    genre: [Shooter]
+    subgenre: []
```
Comment (nikuda) [inline games/a.yaml]: You can remove the subgenre key here.

File: games/a.yaml [hunk 2]
```diff
@@ -250,6 +250,32 @@
...
+  clones:
+  - name: Uebergame
+    repo: https://github.com/Duion/Uebergame
+    url: http://duion.com/games/uebergame/main
+    development: active
+    status: playable
+    lang: [TorqueScript, HLSL, GLSL, C]
+    framework: [Torque 3D]
+    license: MIT
+    content: CC0
```
Comment (nikuda) [inline games/a.yaml]:
```yaml
license: [MIT, CC0]
content:  open
```

Comment (nikuda) [inline games/a.yaml]:
```yaml
license: [MIT, CC0]
content:  open
```

**Pull request merged**